src/event/event.rs

```diff
@@ -58,30 +69,72 @@ impl Event {
     /// Method is available on all platforms, but not all platforms (can) use
     /// this indicator.
     ///
-    /// When HUP events are received is different per platform. For example on
```

Comment:
I'm not sure this documentation is accurate anymore, so I decide to refer to the selector documentation instead. I think it's the best solution as the indicator is too different per platforms. @carllerche do you agree?

Reply:
Yes, I think that is the right decision :+1:

Some wordsmithing, instead of:

> Method is available on all platforms, but not all platforms (can) use this indicator.

Maybe:


> Method is available on all platforms, but not all platforms trigger the HUP event.